packages/cfd/src/Constants/wallet-success-text.tsx

```diff
@@ -1,47 +1,326 @@
+import { CFD_PLATFORMS, Jurisdiction } from '@deriv/shared';
 
 type WalletSuccessContent = {
     title: JSX.Element;
     description: JSX.Element;
     text_submit: JSX.Element;
+    text_cancel?: JSX.Element;
 };
 
 type WalletSuccessContentMapperType = (
     is_demo: boolean,
     account_title: string,
-    mt5_jurisdiction?: string
+    description_title: string,
+    type: 'all' | 'financial' | 'synthetic',
+    is_verified: boolean,
+    manual_status: string,
+    mt5_jurisdiction?: string,
+    platform?: string,
+    currency?: string
 ) => WalletSuccessContent;
 
-const WalletSuccessContentMapper: WalletSuccessContentMapperType = (is_demo, account_title, mt5_jurisdiction) => {
-    let title: JSX.Element, description: JSX.Element, text_submit: JSX.Element, text_cancel: JSX.Element;
-    const TextCancel = <Localize i18n_default_text='Maybe Later' />;
-
-    if (is_demo) {
-        title = <Localize i18n_default_text='Your {{account_title}} account is ready' values={{ account_title }} />;
-        description = (
-            <Localize
-                i18n_default_text='You can now start practicing trading with your {{account_title}} account.'
-                values={{ account_title }}
-            />
-        );
-        text_submit = <Localize i18n_default_text='Continue' />;
-        text_cancel = TextCancel;
-    } else {
-        title = <Localize i18n_default_text='Almost there' />;
-        description = (
-            <Localize
-                i18n_default_text='We need a few minutes to review your documents before you can start trading with your {{mt5_jurisdiction}} account. You’ll get an in-app notification as soon as this is done.'
-                values={{ mt5_jurisdiction }}
-            />
-        );
-        text_submit = <Localize i18n_default_text='Transfer funds' />;
-        text_cancel = TextCancel;
+const WalletSuccessContentMapper: WalletSuccessContentMapperType = (
+    is_demo,
+    account_title,
+    description_title,
+    type,
+    is_verified,
+    manual_status,
+    mt5_jurisdiction,
+    platform,
+    currency
+): WalletSuccessContent => {
+    let title: JSX.Element = <></>;
+    let description: JSX.Element = <></>;
+    let text_submit: JSX.Element = <></>;
+    let text_cancel: JSX.Element | undefined;
+
+    const all_documents_are_ready = is_verified;
+    const need_few_minutes =
+        [Jurisdiction.BVI, Jurisdiction.VANUATU].includes(mt5_jurisdiction) && manual_status !== 'pending';
+    const need_some_days =
+        [Jurisdiction.LABUAN, Jurisdiction.MALTA_INVEST].includes(mt5_jurisdiction) ||
+        ([Jurisdiction.BVI, Jurisdiction.VANUATU].includes(mt5_jurisdiction) && manual_status === 'pending');
+
+    // for DXTrade
+    if (platform === CFD_PLATFORMS.DXTRADE) {
+        // for demo
+        if (is_demo) {
+            title = (
+                <Localize i18n_default_text='Your {{account_title}} demo account is ready' values={{ account_title }} />
```

Comment:
we can remove demo because it is included inside of `account_title`
```suggestion
                <Localize i18n_default_text='Your {{account_title}} account is ready' values={{ account_title }} />
```

Reply:
@aizad-deriv ditto